scripts/testServer.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -e
+PS1="$"
+basedir="$(pwd -P)"
```

Comment:
yeah i thought it was causing me issues with CWD, ill update it to do like the rest tomorrow.